db/seeds.rb

```diff
@@ -5,3 +5,9 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+@hotel_1 = Hotel.create!(name: 'Stanly Hotel', location: 'Estes Park')
+@room_401 = @hotel_1.rooms.create!(rate: 50, suite: 'Room 401')
+@room_402 = @hotel_1.rooms.create!(rate: 50, suite: 'Room 402')
+@guest_1 = Guest.create!(name: 'Jeff', nights: 1)
+GuestRoom.create(guest: @guest_1, room: @room_401)
```

Comment:
I don't think we really need seed data when you're doing an assessment since you'll spend most of your time in the tests and if you wanted to see what the page looks like you can run a `save_and_open_page`.